github/tests/AllTests.py

```diff
@@ -103,6 +103,7 @@
 from BadAttributes import *
 from Equality import *
 from Search import *
+from Retry import *
```

Comment:
This file already has lots of [_wildcard_ imports (from xyz import *)](https://www.python.org/dev/peps/pep-0008/#imports) but do we _need_ to continue that anti-pattern in the case of __Retry__?

Reply:
I believe this is a special case where the wildcard makes sense since this is an aggregation of all test cases. It is more readable and more meaningful to have wildcard imports here, IMO.

Reply:
Understood.